content/docs/next-release/features.md

```diff
@@ -29,10 +29,10 @@ Since v1.35, the Jaeger backend can receive trace data from the OpenTelemetry SD
 ## Multiple storage backends
 
 Jaeger can be used with a growing a number of storage backends:
-* It natively supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+ and Elasticsearch 5.x/6.x/7.x.
+* It natively supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+, Elasticsearch 5.x/6.x/7.x, and OpenSearch 1.0+.
 * It integrates via a gRPC API with other well known databases that have been certified to be Jaeger compliant: [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse).
 * There is embedded database support using [Badger](https://github.com/dgraph-io/badger) and simple in-memory storage for testing setups.
-* There are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB, Logz.io.
+* There are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB, Logz.io. You can find more by following [this issue](https://github.com/jaegertracing/jaeger/issues/638).
```

Comment:
Should Logz.io be in this list? It's a vendor observability product, does it actually work as a standalone database?

Reply:
It can be used as a backend, which was working before they built the full Jaeger solution. So can any backend which supports Elastic API. I will drop it.